packages/xo-server/src/xapi/index.js

```diff
@@ -2348,4 +2348,27 @@ export default class Xapi extends XapiBase {
       )
     }
   }
+
+  async isHyperThreading(hostId) {
+    try {
+      return (
+        (await this.call(
+          'host.call_plugin',
+          this.getObject(hostId).$ref,
+          'hyperthreading.py',
+          'get_hyperthreading',
+          {}
+        )) !== 'false'
```

Comment:
Is it to avoid false negatives as much as possible? Or does it really never return `'true'`? Which would be weird...

Reply:
https://github.com/vatesfr/xen-orchestra/pull/4285#discussion_r295353280